docs-website/src/styles/sphinx.scss

```diff
@@ -1,124 +1,114 @@
-// Styles for Sphinx Python SDK generated docs
-$borderRadius: 5px;
-
-dl.py {
-  margin-bottom: calc(var(--ifm-spacing-vertical) * 2);
-  font-size: 14px;
-  border: 1px solid var(--ifm-hr-border-color);
-  border-radius: $borderRadius;
-
-  code {
-    border: none;
-    background: none;
-  }
-
-  p {
-    margin-bottom: 0;
-  }
-
-  dl {
-    margin-bottom: var(--ifm-spacing-vertical);
-  }
-
-  // The parameter name:
-  em.sig-param > span:first-child {
-    font-weight: bold;
-  }
-
-  > dd:not(:empty) {
-    padding-bottom: var(--ifm-spacing-vertical);
-  }
-
-  dt.sig {
-    box-sizing: border-box;
-    font-size: 0.9rem;
-    padding: var(--ifm-spacing-vertical);
-    border-radius: $borderRadius;
-    font-family: var(--ifm-font-family-monospace);
-    background-color: var(--ifm-background-surface-color);
-  }
-
-  > dd {
-    &:not(:empty) {
-      padding-top: calc(var(--ifm-spacing-vertical) / 2);
-      margin-top: 0;
-      margin-left: var(--ifm-spacing-horizontal);
-      margin-right: var(--ifm-spacing-horizontal);
+.sphinx-api-docs {
```

Comment:
the font size of the headers seems pretty large 
<img width="768" alt="image" src="https://github.com/user-attachments/assets/1485cc26-180f-448e-ab9d-6a9d100541b4" />